requirements.txt

```diff
@@ -1,6 +1,25 @@
-Flask
-cachetools
-gunicorn
-requests
-sodapy
-typing_extensions
\ No newline at end of file
+appdirs==1.4.4
```

Comment:
I don't think all these need to be specified - we can probably just pin the 6 requirements we need?

Reply:
For sure! Done in 9683b58